primitives/version/comparable.go

```diff
@@ -73,3 +73,8 @@ func Equals(a, b common.Version) bool {
 func IsAfter(a, b common.Version) bool {
 	return !IsBefore(a, b) && !Equals(a, b)
 }
+
+// EqualOrAfter returns true a == b or a is after b.
```

Comment:
I feel like we added these kind of methods already and this is a diff that will disappear once we merge base PRs